ts/nni_manager/training_service/reusable/environments/dlcEnvironmentService.ts

```diff
@@ -86,6 +92,8 @@ export class DlcEnvironmentService extends EnvironmentService {
                     environment.setStatus('UNKNOWN');
             }
         });
+        deferred.resolve();
+        return deferred.promise;
```

Comment:
Well, if it's unconditionally resolved, then you can simply use `return Promise.resolve()`.